packages/build/src/core/lingering.js

```diff
@@ -14,7 +14,7 @@ const warnOnLingeringProcesses = async function ({ mode, logs, testOpts: { silen
   const {
     stdout: processList,
   } = await execa(
-    'ps aux | grep -v [p]s | grep -v [g]rep | grep -v [b]ash | grep -v "/opt/build-bin/buildbot" | grep -v [d]efunct | grep -vw \'[build]\' | grep -v "@netlify/build" | grep -v "buildbot.*\\[node\\]" | grep -v "gatsby-telemetry" | grep -v "jest-worker" | grep -v "broccoli-babel-transpiler"',
+    'ps aux | grep -v ps | grep -v grep | grep -v bash | grep -v "/opt/build-bin/buildbot" | grep -v defunct | grep -vw \'[build]\' | grep -v "@netlify/build" | grep -v "buildbot.*\\[node\\]" | grep -v "gatsby-telemetry" | grep -v "jest-worker" | grep -v "broccoli-babel-transpiler"',
```

Comment:
My understanding is that `[` and `]` are passed escaped to `grep`, i.e. this is equivalent to doing this in the shell:

```sh
ps aux | grep -v "[p]s"
```

not:

```sh
ps aux | grep -v [p]s
```

In other words, the brackets are not interpreted by `bash`, but by `grep`. When this happens, they are interpreted as "any characters that is `p`", which is redundant.

I tried it locally:

```js
> (await execa('echo ps | grep [p]s', {shell:'bash'})).stdout
'ps'
> (await execa('echo s | grep [p]s', {shell:'bash'})).stdout
Uncaught Error: Command failed with exit code 1: echo s | grep [p]s
    at makeError (/home/ether/netlify/build/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/home/ether/netlify/build/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (node:internal/process/task_queues:94:5) {
  shortMessage: 'Command failed with exit code 1: echo s | grep [p]s',
  command: 'echo s | grep [p]s',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

Reply:
TIL 😄